src/Compilers/CSharp/Test/Symbol/Symbols/DefaultInterfaceImplementationTests.cs

```diff
@@ -40781,6 +40781,7 @@ static void Main()
             MetadataReference metadataReference = compilation1.EmitToImageReference();
 
             var compilation2 = CreateCompilation(source2, new[] { compilationReference }, options: TestOptions.DebugExe,
+                                                 targetFramework: TargetFramework.NetStandardLatest,
```

Comment:
These changes are necessary because we now correctly report use-site diagnostics on operators, and there were some previously-unreported use-site diagnostics in this scenario. The compilation being referenced uses `NetStandardLatest`, which is netcoreapp3.0, but this compilation uses netstandard2.0, and some use-site diagnostics are reported on the operators for the downgrade.

Reply:
And, to clarify, I don't believe this scenario is observable in practice because the rest of the SDK will be very unhappy with the downgrade :)

---
In reply to: [481307793](https://github.com/dotnet/roslyn/pull/47311#discussion_r481307793) [](ancestors = 481307793)